lib/inc/internal/simple_config_origin.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <hocon/config_origin.hpp>
+#include <hocon/config_value.hpp>
```

Comment:
Yes. Keep header files included in other header files to a minimum, otherwise you really kill compile time.